subsystem/kubeapi_test.go

```diff
@@ -690,6 +746,7 @@ var _ = Describe("[kube-api]cluster installation", func() {
 	})
 
 	AfterEach(func() {
+		printCRs(ctx, kubeClient)
```

Comment:
instead of using it everywhere maybe we can use Init or some other method to define it once?

Reply:
It is currently used just a couple of times, one per each `AfterEach()`, same as DB cleanup, etc. 
the `"[kube-api]cluster installation"` and `"bmac reconcile flow"`test suits are currently indepdandent

Reply:
But once people will add more tests will become a mess no? why not check the deployment type in the aftereach and react according to it? 
All the tests need to clear the DB so why not write it once?

Reply:
maybe something like that https://play.golang.org/p/7_zCj4Lqawr

Reply:
Per discussion @filanov and I just had, I'll be doing this in a separate PR. I Opened [MGMT-8299](https://issues.redhat.com/browse/MGMT-8299) to track this.